net/portmapper/portmapper.go

```diff
@@ -401,18 +420,31 @@ func (c *Client) createOrGetMapping(ctx context.Context) (external netaddr.IPPor
 
 	pmpAddr := netaddr.IPPortFrom(gw, pmpPort)
```

Comment:
just make this variable be called pxpAddr and use it for both. then ditch pcpAddr{,u}